res/values-tr/strings.xml

```diff
@@ -46,13 +46,13 @@
     <string name="PLAYING">"OYNANIYOR"</string>
     <string name="MULTIPLAYER">"ÇOK OYUNCULU"</string>
 
-    <string name="JOIN">"KATIL"</string>
+    <string name="JOIN">"KATILMAK"</string>
```

Comment:
This word has an infinitive suffix
Katıl this word has already been translated correctly